esignet-default.properties

```diff
@@ -48,7 +48,7 @@ mosip.esignet.header-filter.paths-to-validate={'${server.servlet.path}/authoriza
 
 
 #mosip.esignet.captcha.required.auth-factors={'PWD'}
-mosip.esignet.captcha.required.auth-factors={}
+mosip.esignet.captcha.required=
```

Comment:
Let us set the default value to 'send-otp,pwd' @pvsaidurga

Reply:
ok anusha.